front/tests/pathfinding-local.spec.ts

```diff
@@ -0,0 +1,88 @@
+      playwrightHomePage = new PlaywrightHomePage(page);
+      playwrightScenarioPage = new PlaywrightScenarioPage(page);
+
+      await playwrightHomePage.goToHomePage();
+
+      // Real click on project, study, scenario
+      await playwrightHomePage.goToOperationalStudiesPage();
+      await playwrightHomePage.page
+        .getByTestId('_@Test integration project')
+        .locator('div')
+        .getByRole('button')
+        .click();
+
+      await playwrightHomePage.page
+        .getByTestId('_@Test integration study')
+        .getByRole('button')
+        .click();
+
+      await playwrightHomePage.page
+        .getByTestId('_@Test integration scenario')
+        .getByRole('button')
+        .click();
+      await playwrightScenarioPage.checkInfraLoaded();
+      await playwrightHomePage.page.getByTestId('scenarios-add-train-schedule-button').click();
+    });
+
+    test('Testing pathfinding electrification error', async () => {
+      // ***************** Rolling Stock *****************
+      const playwrightRollingstockModalPage = new PlaywrightRollingstockModalPage(
+        playwrightHomePage.page
+      );
+
+      await playwrightRollingstockModalPage.openRollingstockModal();
+      await playwrightRollingstockModalPage.searchRollingstock(VARIABLES.searchRollingstock);
+      const rollingstockCard = playwrightRollingstockModalPage.getRollingstockCardByTestID(
+        VARIABLES.rollingstockTestID
+      );
+      await rollingstockCard.click();
+      await rollingstockCard.locator('button').click();
+
+      // ***************** Composition Code *****************
+      await playwrightScenarioPage.openTabByText('Paramètres de simulation');
+      await playwrightScenarioPage.getSpeedLimitSelector.click();
+      await playwrightScenarioPage.getSpeedLimitSelector.locator('input').fill('Haut');
+      await playwrightScenarioPage.getSpeedLimitSelector
+        .getByRole('button', { name: 'Haut le pied' })
+        .click();
+
+      // ***************** Choice Origin/Destination *****************
+      const playwrightMap = new PlaywrightMap(playwrightHomePage.page);
+      await playwrightScenarioPage.openTabByText('Itinéraire');
+
+      // Search and select origin
+      await playwrightMap.openMapSearch();
+      await playwrightMap.searchStation('quimper');
+      await playwrightHomePage.page.getByRole('button', { name: 'QR Quimper SP 87474098' }).click();
+      await playwrightMap.closeMapSearch();
+      await playwrightMap.page.waitForTimeout(200);
+      await playwrightMap.clickOnMap({ x: 230, y: 165 });
+      await playwrightMap.clickOnOrigin();
+
+      // Search and select destination
+      await playwrightMap.openMapSearch();
+      await playwrightMap.searchStation('brest');
+      await playwrightHomePage.page.getByRole('button', { name: 'BT Brest 87474007' }).click();
+      await playwrightMap.closeMapSearch();
+      await playwrightMap.page.waitForTimeout(200);
```

Comment:
Are all these timeouts for map loading ? If so, it might be good to have a common timeout value (500?), and maybe encase the timeout in a method `addTimeoutForMapToLoad` or something.

Reply:
Done